modules/riemann/riemann-grammar.ym

```diff
@@ -114,13 +115,13 @@ riemann_option
           {
             riemann_dd_set_field_tags(last_driver, $3);
           }
-        | KW_TYPE '(' LL_IDENTIFIER riemann_type_options ')'
+        | KW_TYPE '(' LL_IDENTIFIER riemann_tls_options ')'
           {
             CHECK_ERROR(riemann_dd_set_connection_type(last_driver, $3), @3,
                         "Unknown Riemann connection type: %s", $3);
             free($3);
           }
-        | KW_TYPE '(' LL_STRING riemann_type_options ')'
+        | KW_TYPE '(' LL_STRING riemann_tls_options ')'
           {
```

Comment:
btw, there's a rule called "string" that maps either LL_IDENTIFIER or LL_STRING, so the two rules above can be collapsed into a single one.